ios/StripeTerminalReactNative.swift

```diff
@@ -420,7 +427,36 @@ class StripeTerminalReactNative: RCTEventEmitter, DiscoveryDelegate, BluetoothRe
             return
         }
 
-        Terminal.shared.createPaymentIntent(paymentParams) { pi, error in
+        let offlineBehavior = params["offlineBehavior"] as? String
+        let offlineModeTransactionLimit = params["offlineModeTransactionLimit"] as? NSNumber ?? 0
```

Comment:
this is not an SDK param, we don't need to worry about it in the React Native or native code. Integrations can define and check it themselves in their app before proceeding with an offline transaction